### ISSUE TITLE
cannot call emscripten method Mat.rows getter on deleted object

### ISSUE THREAD

**Issue by EasonC13** [Piemontez/opencvflow#1]
Hi
I'm new here to try opencvflow

I'm here to report that when I [followed the documentation](https://opencvflow.org/docs/v_0/components/100-io) to build a flow like the following photo, I face an error.

<img width="527" alt="image" src="https://user-images.githubusercontent.com/43432631/162095593-55467f56-770e-493f-9b1b-6714f42e4461.png">

```
cannot call emscripten method Mat.rows getter on deleted object
```

Thank you

P.S: This will be a great tool, I love the idea of it.

**Comment by Piemontez** [comment]
Thanks.

In that moment i will providing same validations for the system to inform which connections are allowed.

In your case i will need to improve the garbage collector to didn't remove same image mat dates when this in use.

**Comment by Piemontez** [comment]
Image file loader fixed on version [0.1.3](https://github.com/Piemontez/opencvflow/releases/tag/0.1.3)

Please pull main branch if you are using the local version, or upgrade to version 0.1.3
